presto-main/etc/catalog/druid.properties

```diff
@@ -0,0 +1,3 @@
+connector.name=druid
+druid.coordinator-url=http://localhost:8081
+druid.broker-url=http://localhost:8082
```

Comment:
I think it's fine to add extra configurations here for development purposes. We may also need to be careful not to let these configs become legacy code in the future.